tests/HostTest.php

```diff
@@ -44,18 +44,38 @@ public function testDoesNotHaveAnySubdomain()
         $this->assertFalse($h->hasSubdomain());
     }
 
-    public function testAddSubdomain()
+    public function dataForTestAddSubDomains()
+    {
+        return [
+            ['www'],
+            ['70684e54'],
```

Comment:
in this case regardless of the host before calling add the result would be greater than 253.
`strlen('www.example.com') . ".70684e54"` would be `15.70684e54` or `9223372036854775807`